### ISSUE TITLE
Companies in the wrong topic area - IT

### ISSUE THREAD

**Issue by kmonterr** [18F/nsf-sbir#1039]
SensorHound and Runtime Verification as featured companies in Rick’s topic (IoT), whereas both are in the IT topic.

**Comment by seakhtar** [comment]
Hi @kmonterr -

These should be showing up in the correct place- if it looks okay, I'll push to production:

https://federalist-proxy.app.cloud.gov/preview/18f/nsf-sbir/companies-fix/portfolio/

**Comment by kmonterr** [comment]
Yes! I see SensorHound in IT, but Runtime Verification isn't showing up at all. Maybe they aren't an active Phase II anymore?

**Comment by seakhtar** [comment]
I think it might be because IT has quite a few featured companies now and the display cuts them off at 9. We could take some of the other companies off if the Runtime one is important to show?

**Comment by kmonterr** [comment]
ohhhh! Not a big deal!  Thanks, Sarah!
